src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/SendNeedsApprovalReminderServiceTest.kt

```diff
@@ -38,6 +39,17 @@ class UnapprovedLicenceServiceTest {
 
   @Test
   fun `service sends an email`() {
```

Comment:
In order to mock the return from the query, as it is an interface, it needs to be done as an object. This method is implementing the interface anonymously by using the `object` keyword.